### ISSUE TITLE
[Cygwin] Python plugin doesn't load on Weechat 2.0

### ISSUE THREAD

**Issue by Yavor-Ivanov** [weechat/weechat#1119]
OS: Windows 7 Ultimate 64bit
Platform: Cygwin 2.9.0(0.318/5/3)
Weechat version output:  `WeeChat 2.0 [compiled on Dec  3 2017 20:54:02]`

I've installed `weechat-python` via `apt-cyg` and tried running `/plugin load python`. I get this error message:
```
weechat =!= | Error: unable to load plugin "/usr/lib/weechat/plugins/python.dll": No such processugins
weechat =!= | If you're trying to load a script and not a C plugin, try command to load scripts (/perl, /python, ...)
```

Yet, it appears the plugin is installed:
```
$ file /usr/lib/weechat/plugins/python.dll
/usr/lib/weechat/plugins/python.dll: PE32+ executable (DLL) (console) x86-64, for MS Windows

```

**Comment by flashcode** [comment]
Hi,
I have no such problems with python plugin, but I installed packages via setup.exe (not sure if this is really different from apt-cyg).
I just released the version 2.0.1 of WeeChat, so I will build Cygwin packages soon (tomorrow). Once they are built, you could test with this version.

**Comment by flashcode** [comment]
Hi,
The version 2.0.1 is now available in Cygwin, could you please try with this version?

**Comment by Yavor-Ivanov** [comment]
I backed up my cygwin `bin` folder, and upgraded Weechat from the cygwin installer, instead of from `apt-cyg` this time. Unfortunately, the upgrade broke python, so I just copied the python binaries from the backup.

Weechat 2.0.1 loads python fine now. The bad news is that I have _no idea_ what fixed it: the new version you pushed, anything else that could have been changed from the re-install, or the fact that I have all new cygwin binaries except for pre-upgrade python version. :man_shrugging:
